src/main/java/com/github/codedoctorde/itemmods/api/block/CustomBlock.java

```diff
@@ -88,7 +88,7 @@ else if (dropType == BlockDropType.FORTUNE)
         if (event.isCancelled())
             return false;
         Location dropLocation = getBlock().getLocation().clone().add(0.5, 0, 0.5);
-        drops.forEach(drop -> Objects.requireNonNull(dropLocation.getWorld()).dropItemNaturally(dropLocation, drop));
+        event.getDrops().forEach(drop -> Objects.requireNonNull(dropLocation.getWorld()).dropItemNaturally(dropLocation, drop));
```

Comment:
That doesn't work when it cancel this event

Reply:
That forEach will be never triggered because on event cancelled the method returns false.